### PR TITLE
Make IPAddress field for struct IPAddresses to array

### PR DESCRIPTION
**Pull request by slavaavr**
https://github.com/vmware/go-vcloud-director/issues/349

### CODE REVIEW

**Review by Didainius — APPROVED** [review]
Thanks for this fix. I am approving it, but have one more ask.
Can you add a changelog note to the top of `CHANGELOG.md`. (replacing **** with your fix comment)



```
## 2.11.0 (Unreleased)

BUGS FIXED:
* **** [#350](https://github.com/vmware/go-vcloud-director/pull/350)
```

Thanks!

File: types/v56/types.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ type NetworkFeatures struct {
 // Description: A list of IP addresses.
 // Since: 0.9
 type IPAddresses struct {
-	IPAddress string `xml:"IpAddress,omitempty"` // An IP address.
+	IPAddress []string `xml:"IpAddress,omitempty"` // An IP address.
```
Comment (dataclouder) [inline types/v56/types.go:138]:
```suggestion
	IPAddress []string `xml:"IpAddress,omitempty"` // A list of IP addresses.
```

**Comment by slavaavr** [comment]
@Didainius, did it!

**Review by dataclouder — APPROVED** [review]

**Comment by Didainius** [comment]
Thanks @slavaavr for the fix. I have tagged the code after your merge with `2.11.0-alpha.1` for convenience.

**Pull request merged**